reaktive/src/commonMain/kotlin/com/badoo/reaktive/observable/FlatMapCompletable.kt

```diff
@@ -3,21 +3,14 @@ package com.badoo.reaktive.observable
 import com.badoo.reaktive.completable.Completable
 import com.badoo.reaktive.completable.asObservable
 
-/**
- * Calls the [mapper] for each element emitted by the [Observable] and subscribes to the returned inner [Completable].
- * All inner [Completable]s are subscribed concurrently without any limits.
- *
- * Please refer to the corresponding RxJava [document](http://reactivex.io/RxJava/javadoc/io/reactivex/Observable.html#flatMapCompletable-io.reactivex.functions.Function-).
- */
-fun <T> Observable<T>.flatMapCompletable(mapper: (T) -> Completable): Completable =
-    flatMapCompletable(maxConcurrency = Int.MAX_VALUE, mapper = mapper)
-
 /**
  * Calls the [mapper] for each element emitted by the [Observable] and subscribes to the returned inner [Completable].
  * The maximum number of concurrently subscribed inner [Completable]s is determined by the [maxConcurrency] argument.
  *
+ * All inner [Completable]s are subscribed concurrently without any limits.
```

Comment:
Add "By default, " here and for other variants?

Reply:
Done